Directory.Build.targets

```diff
@@ -87,11 +87,15 @@
       <TargetingPackVersion Condition="'$(IsServicingBuild)' != 'true'">$(MicrosoftNETCoreAppPackageVersion)</TargetingPackVersion>
     </KnownFrameworkReference>
 
+    <KnownFrameworkReference Condition="'$(UseAspNetCoreSharedRuntime)' != 'true'" Remove="Microsoft.AspNetCore.App" />
+    <KnownFrameworkReference Remove="Microsoft.WindowsDesktop.App" />
+
     <KnownFrameworkReference Update="Microsoft.AspNetCore.App">
```

Comment:
Seems to be working but I'm surprised this doesn't need `Condition="'$(UseAspNetCoreSharedRuntime)' == 'true'"`. (As-is, there's nothing to update and I thought that was an error case.)